gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FileExporter.java

```diff
@@ -94,6 +100,15 @@ public String saveVoltageMonitoringResultInJson(VoltageMonitoringResult result,
         return minioAdapter.generatePreSignedUrl(voltageResultPath);
     }
 
+    private String zipTargetNameToJsonName(String targetName) {
+        if (StringUtils.isNotBlank(targetName) &&
+                (targetName.contains(".ZIP") || targetName.contains(".zip"))) {
```

Comment:
can you replace those 2 conditions with "targetName.toLowercase().contains(".zip")" or it wont work ?